src/Symfony/Component/HttpClient/Internal/CurlClientState.php

```diff
@@ -23,8 +23,8 @@
  */
 final class CurlClientState extends ClientState
 {
-    /** @var \CurlMultiHandle|resource */
-    public $handle;
+    /** @var array<\CurlMultiHandle|resource> */
+    public $handle = [];
```

Comment:
I would name the property `handles`, as it has several ones

Reply:
renamed